cp-remote

```diff
@@ -335,12 +336,34 @@ function port_forward {
     kubectl --context="$(context)" --namespace="$(namespace)" port-forward "$POD" "$LOCAL_PORT":"$REMOTE_PORT"
 }
 
+function check_version {
+
+    CURRENT_VERSION=`cat ${0} | grep "^# Version" | head -1 | awk '{print $3}'`
+    LATEST_VERSION=`curl -s https://raw.githubusercontent.com/continuouspipe/remote-environment-client/gh-pages/downloads/cp-remote-latest | grep "# Version" | awk '{print $3}'`
```

Comment:
You can also use the `manifest` file if it can help... 
```
https://raw.githubusercontent.com/continuouspipe/remote-environment-client/gh-pages/manifest.json
```